tests/cypress/e2e/unit_tests/upgrade_fleet.spec.ts

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright © 2023 - 2025 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import 'cypress/support/commands';
+import { qase } from 'cypress-qase-reporter/dist/mocha';
+
+export const branch = "master";
+export const fleetLocal = 'fleet-local'
```

Comment:
Why this constant is needed here?
We are using `fleet-local` in other tests as well and no issues.

Reply:
I thought it could be nice to store string in variable and use it in various place. But it's okay, I will remove it.

Reply:
Done.